script/todo.php

```diff
@@ -27,6 +27,7 @@ public function load(string $path)
 
     public function store(string $path, string $content): void
     {
+        $path = $this->realpath($path);
```

Comment:
Remove this line.

Its problem is that it calls the realpath method. If we delete the todo.md file, the method will return `null` and an error will occur, because the path parameter of the `file_put_contents` function cannot be null.

See https://github.com/Laravel-Lang/lang/pull/1401/commits/deb228567cec25e33eb99d8db64a2557b3abf437

Reply:
thx